src/chart-review/chart-review.component.tsx

```diff
@@ -12,8 +12,11 @@ import styles from "./chart-review.css";
 import { useConfig } from "@openmrs/esm-module-config";
 
 import { getView, View } from "../view-components/view-utils";
-import { ChartConfig, Navbar } from "../root.component";
 import { VisitButton } from "@openmrs/esm-patient-chart-widgets";
```

Comment:
@brandones just to bring to your attention that I think it's inevitable that patient-chart will be tightly coupled to the patient-chart-widgets. As you can see the import happens here for this VisitButton, and I think we are going to see more of this as the app grows.

Reply:
Well, for one, the system for managing visits in the patient chart was never really designed, so I would expect it to change.

And two, I think this is an implementation mistake. These Visit* things are not widgets. They are components of the patient chart. They should be part of the patient chart.

I'm confident we can maintain this decoupling, and that doing so will keep the code cleaner and the system easier to understand, maintain, and extend.

Reply:
I agree. We need distinguish the two very clearly. The documentation will go a long way in helping with it. It also needs to come out very clearly why the two modules should not be tightly coupled, and the trade-offs involved. Until now, I thought it was intentional that patient-chart and patient-chart-widget are tightly coupled, but managed in different repos to make it possible/encourage these widgets to be re-used and extended in 3rd party patient-chart-widgets modules. I hope we continue this discussion about the current design, because as you can see it is easy to mis-understand the design, and can get out of hand real quick. 
That said, what's pending right now? I was hoping to address the issue of loading configs for 3rd party configs in this PR.

Reply:
Continuing the conversation on Slack....